index.html

```diff
@@ -128,6 +128,11 @@
 <pre><code>./run-tests.sh</code></pre>
 
 <h2>
+<a name="api-sparkr" class"anchor" href="#api-sparkr"><span class="octicon octicon-link"></span></a>SparkR package API documentation</h2>
+
+<a href="rdocs/00Index.html">Documentation for package ‘SparkR’</a>
```

Comment:
Instead of `00Index.html` I actually think `index.html` looks better ? Also I guess we should say this is for Spark 1.2 but we can do that when we add docs for Spark 1.3